client/src/charts/wrapped_nivo/nivo_line.js

```diff
@@ -152,13 +167,13 @@ export class NivoResponsiveLine extends React.Component {
             motionDamping,
             motionStiffness,
           }}
-          legends={ IE_fixed_legends }
+          legends={ fix_legend_symbols(legends) }
           tooltip={ (d) => tooltip( d, get_formatter(is_money, text_formatter, false) ) }
```

Comment:
maybe change the `d` variable to be a slight bit more explicit

Reply:
Ehh, can't really think of one, maybe `datum`? It's a vague D3 wrapped chunk of graph data, with a bunch of other stuff crammed in there. The D3 style (which I don't love) is to just call that `d` (I feel like that betrays D3's rough edges, that they spend so much time dealing with messy variables that can't be easily named).